compiler/damlc/tests/bond-trading/Setup.daml

```diff
@@ -76,46 +81,46 @@ template HelperSetupJob
       controller party
       do create Helper with party
 
-marketSetupEntryExample party = do
-  party1 <- getParty party
+marketSetupEntryExample party = script do
+  party1 <- allocateParty party
   return MarketSetupEntry with
     party = party1
     bondEntries = [ BondEntry with isin = "1234"; amount = 100.0 ]
     cashEntries = [ CashEntry with currency = "USD"; amount = 100.0 ]
 
-marketSetupJobExample = do
-  acme <- getParty "Acme Bank"
+marketSetupJobExample acme = script do
   entries <- mapA marketSetupEntryExample ["party1", "party2", "party3", "party4", "party5"]
   return MarketSetupJob with
     issuer = acme
     entries
 
 
-setupExample = scenario do
-    passToDate $ date 2018 May 14
+setupExample = script do
+    setDate 2018 May 14
 
-    acme <- getParty "Acme Bank"
+    acme <- allocateParty "Acme Bank"
 
-    job <- marketSetupJobExample
-    submit acme do create job
+    job <- marketSetupJobExample acme
+    submit acme do createCmd job
```

Comment:
All other commands were rewritten with inline `submit`, should we change this one too?

Reply:
Ah oops, I must have missed this